### PR TITLE
Add Request and Response to snippet

### PR DESCRIPTION
**Pull request by grongierisc**
This PR add Request and Response message type to snippet.
Also, add Snippet shortcut to Class for Class Request/Response/Operation/Service/Production

Add TryCatch snippet and ThrowOnError

### CODE REVIEW

**Review by daimor — DISMISSED** [review]

File: snippets/objectscript-class.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "description": "Relationship"
   },
   "BusinessService": {
-		"prefix": ["BusinessService","Interoperability"],
+		"prefix": ["BusinessService","Interoperability","ClassOperation"],
```
Comment (gjsjohnmurray) [inline snippets/objectscript-class.json:73]: Did you mean `ClassService` here?

File: snippets/objectscript-class.json [hunk 2]
```diff
@@ -84,7 +84,7 @@
 		"description": "Business Service Definition"
 	},
 	"BusinessOperation": {
-		"prefix":["BusinessOperation","Interoperability"],
+		"prefix":["BusinessOperation","Interoperability","ClassService"],
```
Comment (gjsjohnmurray) [inline snippets/objectscript-class.json:87]: `ClassOperation`?

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]

File: snippets/objectscript-class.json (hunk 1 above)
Comment (grongierisc) [reply snippets/objectscript-class.json]: Thanks for this review, you are right, PR updated

File: snippets/objectscript-class.json (hunk 2 above)
Comment (grongierisc) [reply snippets/objectscript-class.json]: Thanks for this review, you are right, PR updated

**Review by gjsjohnmurray — APPROVED** [review]

**Pull request merged**